### PR TITLE
REF/TST: method-specific files for DataFrame timeseries methods

### PR DESCRIPTION
**Pull request by jbrockmendel**
xref #32226.

### CODE REVIEW

**Review by simonjayhawkins — APPROVED** [review]
@jbrockmendel lgtm

File: pandas/tests/frame/conftest.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 import numpy as np
 import pytest
 
 from pandas import DataFrame, NaT, date_range
 import pandas._testing as tm
 
 
+@pytest.fixture(params=product([True, False], [True, False]))
+def close_open_fixture(request):
+    return request.param
```
Comment (simonjayhawkins) [inline pandas/tests/frame/conftest.py]: if this does not have utility beyond the one test in pandas/tests/frame/methods/test_between_time.py, I would remove in a follow-up

**Comment by simonjayhawkins** [comment]
Thanks @jbrockmendel

File: pandas/tests/frame/conftest.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/frame/conftest.py]: agreed

**Pull request merged**